src/codeunit/SpotifyService/ITILoadPlaylist.Codeunit.al

```diff
@@ -0,0 +1,63 @@
+codeunit 50106 "ITI Load Playlist"
+{
+    trigger OnRun()
+    begin
+
+    end;
+
+    procedure LoadPlaylistsFromUsersAccount()
```

Comment:
W tej funkcji ale i w innych w tym obiekcie używasz słowa user. Sugeruje to trochę jakbyśmy mogli mieć zdefiniowane X userów i udostępniamy dla nich funckjonalność pobierania playlist. A w rzeczywistości to zawsze używamy tego Twojego usera, więc dla czytelności proponuję usunąć wszędzie słowo user